src/__tests__/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+  const { asFragment } = render(<App />);
+
+  it("renders correctly", () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
```

Comment:
I would recommend adding more tests than just snapshots. You could add test to:
- check if the NASA logo is rendered correctly, and as the correct alt tag
- check that your search box renders correctly + the search button image attributes
- check your search functionality works correctly
- check that your search Results are being displayed correctly